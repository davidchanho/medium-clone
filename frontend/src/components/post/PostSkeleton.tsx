import React from "react";
import Skeleton from "react-loading-skeleton";

function PostSkeleton() {
  return (
    <div>
      <Skeleton count={5} />
      <Skeleton width={200} height={135} />
    </div>
  );
}

export default PostSkeleton;
