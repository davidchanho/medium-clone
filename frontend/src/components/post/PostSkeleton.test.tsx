import React from "react";
import renderer from "react-test-renderer";
import PostSkeleton from "./PostSkeleton";

describe("PostSkeleton", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<PostSkeleton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
