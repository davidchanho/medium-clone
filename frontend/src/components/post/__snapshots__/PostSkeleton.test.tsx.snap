// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostSkeleton renders correctly 1`] = `
<div
  className="my-2 d-flex"
>
  <span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={Object {}}
    >
      ‌
    </span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={Object {}}
    >
      ‌
    </span>
  </span>
  <span>
    <span
      className="react-loading-skeleton css-1eg0sfo-skeletonStyles-Skeleton"
      style={
        Object {
          "height": 135,
          "width": 200,
        }
      }
    >
      ‌
    </span>
  </span>
</div>
`;
